source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/inputhandler/CheckBoxSetInputHandler.java

```diff
@@ -17,9 +18,9 @@
 
 public class CheckBoxSetInputHandler extends BaseInputHandler
 {
-    public CheckBoxSetInputHandler(BaseInputElement baseInputElement, List<CheckBox> checkBoxList)
+    public CheckBoxSetInputHandler(BaseInputElement baseInputElement, List<CheckBox> checkBoxList, RenderedAdaptiveCard renderedAdaptiveCard, long cardId)
```

Comment:
BaseInputHandler has 2 constructors. Others don't have. Its better to make it mandatory for them to pass all the parameters. Otherwise their ValueChangedAction property will not work and that will come later as a bug.